wezterm-gui/src/termwindow/mod.rs

```diff
@@ -928,6 +928,20 @@ impl TermWindow {
                 }
                 Ok(true)
             }
+            WindowEvent::DroppedFile(paths) => {
+                let pane = match self.get_active_pane_or_overlay() {
+                    Some(pane) => pane,
+                    None => return Ok(true),
+                };
+                let paths = paths
+                    .iter()
+                    .map(|path| format!("{:?}", path.to_string_lossy().to_string()))
+                    .collect::<Vec<_>>()
+                    .join(" ");
+                pane.writer().write_all(paths.as_bytes())?;
```

Comment:
You can send it as a paste this way:

```suggestion
                pane.trickle_paste(paths)?;
```

Reply:
Nice way.